src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -113,6 +118,52 @@ export default {
 		menuOpen: {
 			type: Boolean,
 			default: false
+		},
+		/**
+		 * Refers to the icon on the left, this prop accepts a class
+		 * like 'icon-category-enabled' or an absolute URL. If you
+		 * pass in an URL the component will display the linked image;
+		 */
+		icon: {
+			type: String,
+			required: false
+		},
+		/**
+       	* Allows you to choose wether you want a parent element to always
+	   	* display it's children (true) or give the possibility to
+		* collapse them into one element (false).
+       	*/
+		forceExpand: {
+			type: Boolean,
+			default: true
+		},
+		/**
+		* Passing in a HEX color string in this prop will hide any
+		* icon or image and display a bullet with the selected
+		* colors instead.
+       	*/
+		bullet: {
+			type: String,
+			default: ''
+		},
+		isEditable: {
+			type: Boolean,
+			default: false
+		},
+		/**
+		* Displays a loading animated icon on the left of the element.
+       	*/
+		isLoading: {
+			type: Boolean,
+			default: false
+		},
+		/**
+		* Gives the possibility to pass in an array of additional classes
+		* that will be applied to the root element.
+       	*/
+		classes: {
```

Comment:
The classes prop doesn't make sense, you can directly add classes on the component iself :)

Reply:
Ok!